pkg/api/error.go

```diff
@@ -95,6 +95,8 @@ var (
 	CloudErrorCodeQuotaExceeded                      = "QuotaExceeded"
 	CloudErrorResourceProviderNotRegistered          = "ResourceProviderNotRegistered"
 	CloudErrorCodeJMESPathSearchFailed               = "JMESPathSearchFailed"
+	CloudErrorRequestDisallowedByPolicy              = "RequestDisallowedByPolicy"
```

Comment:
I'd think we should reuse CloudErrorCodeDeploymentFailed instead of adding CloudErrorRequestDisallowedByPolicy - the end user should be able to tell the nested error code as they can in deployARMTemplate.

Reply:
Changed